### DISCUSSION TITLE
Getting Started with Seeed Studio Wiki Platform | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1844] [Q&A]
# Getting Started with Seeed Studio Wiki Platform | Seeed Studio Wiki

Being a trusted Internet of Things (IOT) hardware partner since 2008, Seeed Studio is dedicated to realizing interactive digital replica of our physical world. This website is an open source platform with a collection of all wikis currently published by Seeed Studio and updated weekly to showcase our product range and research, which includes digitalisation of the natural environment.

https://wiki.seeedstudio.com/Getting_Started/

**Comment by mohammad508** [comment]
i use ReSpeaker 4-Mic Array for Raspberry Pi in kernal 5.4.83-v7+ but kit respeaker doest not install driver in this kernal, it show this 
pi@pi:~ $ arecord -L
null
    Discard all samples (playback) or generate zero samples (capture)
jack
    JACK Audio Connection Kit
pulse
    PulseAudio Sound Server
default
    Playback/recording through the PulseAudio sound server
playback
ac108
usbstream:CARD=Headphones
    bcm2835 Headphones
    USB Stream Output
pi@pi:~ $

**Comment by dennisct** [comment]
I have the same problems. Cannot get it working
